RELEASENOTES-1.4.docu

```diff
@@ -257,4 +257,8 @@
       now by default log one <tt>spec-violation</tt> message on log level 1
       on first such write, and no logging on further such writes.
   </add-note></build-id>
+  <build-id value="next"><add-note> Added simple C99-style designated
+      initializers. Use together with <tt>struct</tt>, <tt>extern struct</tt>,
```

Comment:
The release notes have refmanual-level precision, which hampers readability. Better to give a simple example of what you can do with it.

Reply:
True. Examples feel redundant, but I could say "struct-like" types instead of listing all the damn types.

Reply:
An example is a polite way to compensate that not everyone knows the term 'designated initializer'. Most of our users are HW-centric coders, not language geeks.

Reply:
Example added.